pkg/config/config_test.go

```diff
@@ -36,7 +36,7 @@ const testTemplateWithEnvVar = "Follow the {{.color}} {{ .Env.ANIMAL }}"
 var testDevEnvironment = environment.NewEnvironment("development", "Dev", "", "https://url/to/dev/environment", "DEV")
 var testHardeningEnvironment = environment.NewEnvironment("hardening", "Hardening", "", "https://url/to/hardening/environment", "HARDENING")
 var testProductionEnvironment = environment.NewEnvironment("prod-environment", "prod-environment", "production", "https://url/to/production/environment", "PRODUCTION")
-var testManagementZoneApi = api.NewApi("management-zone", "/api/config/v1/managementZones")
+var testManagementZoneApi = api.NewApi("management-zone", "/api/config/v1/managementZones", "values")
```

Comment:
I guess we only need it for testing, but something like an `api.NewStandardApi(id, path)` -> api.NewApi(id,path,"values") would remove the need to explicitly call it with `"values"` everywhere. 
(makes you notice that method overloading or default values are pretty nice convenience features...)

Reply:
Good idea! Yeah, method overloading would be nice to have. :smile: